CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Updates should follow the [Keep a CHANGELOG](http://keepachangelog.com/) princip
 
 ## [Unreleased][unreleased]
 
+#### Added
+* Added ApiLink Macro
```

Comment:
pls uprav tak aby to bolo konzistentne so zvyskom suboru

Reply:
neviem co myslis? pridat entery?

Reply:
jj naformatovat to tak ako su ostatne